README.md

```diff
@@ -23,3 +23,114 @@ All of the following exceptions defined in exceptions in this package have been
 4. `UnknownApiException` -> `UnknownApiError`
 5. `NoInternetException` -> `NoInternetError`
 6. `UnstableInternetException` -> `UnstableInternetError`
+
+## Introduction and Usage
+
+Fa_Flutter_API_Client is a customised implementation of API_client. It serves as a base for other FieldAssist apps.
+This package is used to implemented Dio effortlessly into other FieldAssist apps.
+
+The basic functionalities that this package offers are:-
```

Comment:
Reformat this para. It seems new line missing when reading. Please improve readability

Reply:
Is it good now?